locales/en/settings.ftl

```diff
@@ -57,7 +57,7 @@ settings-deactivate-account-info = You can deactivate { -product-short-name } by
 settings-deactivate-account-info-2 = You can deactivate { -product-short-name } by deleting your { -brand-mozilla-account }.
 # Deprecated
 settings-fxa-link-label = Go to { -brand-firefox } Settings
-settings-fxa-link-label-2 = Go to { -brand-mozilla } Settings
+settings-fxa-link-label-3 = Go to { -brand-mozilla } account settings
```

Comment:
Wouldn’t we use `-brand-mozilla-account` here?

Reply:
![image](https://github.com/mozilla/blurts-server/assets/4251/a41b95b3-878c-49a9-8f80-3f58ae87e95c)

0fff35b24